src/cdn_definitions/_impl/__init__.py

```diff
@@ -43,15 +49,15 @@ def __init__(self, **kwargs):
 
 def rhui_aliases():
     """Returns:
-        list[:class:`~PathAlias`]
-            A list of aliases relating to RHUI paths.
+    list[:class:`~PathAlias`]
```

Comment:
Have you verified that this displays correctly on the rendered docs?

The indent in general is meaningful.  This commit is changing the string from:

```
Returns:
    list[:class:`~PathAlias`]
        A list of aliases relating to RHUI paths.
```

to:

```
Returns:
list[:class:`~PathAlias`]
    A list of aliases relating to RHUI paths.
```

Doc strings are processed via this tool: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

And in the examples there you can see that the docs for return value are always indented under "Returns", so I have a doubt whether this change is correct.
I'm not sure if the doc string parser is still going to process this the same way as it did before...

Reply:
Is it `black` tool reformatting this?

Reply:
I see now there's a separate commit in the PR saying that it's coming from black.

I checked this locally and I'm a bit confused actually, as both before and after this change sphinx generates warnings:

```
/home/rmcgover/src/cdn-definitions/.tox/docs/lib/python3.8/site-packages/cdn_definitions/_impl/__init__.py:docstring of cdn_definitions._impl.rhui_aliases:3: WARNING: Unexpected indentation.
/home/rmcgover/src/cdn-definitions/.tox/docs/lib/python3.8/site-packages/cdn_definitions/_impl/__init__.py:docstring of cdn_definitions._impl.origin_aliases:3: WARNING: Unexpected indentation.
```

So maybe it was not correct either before or after?  However, the rendered docs seem OK to me, so no objections to this.